app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -190,6 +190,9 @@ class TabTrayView(
                 } else {
                     view.context?.getString(R.string.no_open_tabs_description)
                 }
+                view.tabsTray.asView().isVisible = false
```

Comment:
might be cleaner to just do this line with `isVisible = hasNoTabs` next to the `tab_tray_empty_view`. Up to you!